src/main/kotlin/no/nav/familie/ba/sak/kjerne/autovedtak/satsendring/AutovedtakSatsendringService.kt

```diff
@@ -95,9 +95,7 @@ class AutovedtakSatsendringService(
         }
 
         if (harUtbetalingerSomOverstiger100Prosent(sisteIverksatteBehandling)) {
-            satskjøringForFagsak.feiltype = "UTBETALING_OVER_100_PROSENT"
-            satskjøringRepository.save(satskjøringForFagsak)
-            return SatsendringSvar.FANT_OVER_100_PROSENT_UTBETALING
+            logger.warn("Det løper over 100% utbetaling på fagsak=${sisteIverksatteBehandling.fagsak.id}")
```

Comment:
Tenker det er lettere å søke i Kibana hvis man skriver prosent i stedet for %
```suggestion
            logger.warn("Det løper over 100 prosent utbetaling på fagsak=${sisteIverksatteBehandling.fagsak.id}")
```

Reply:
Fiksa i [% -> prosent](https://github.com/navikt/familie-ba-sak/pull/3396/commits/b736688ee4f44be92a4802bc91042f10e4bde45f)